mkosi

```diff
@@ -30,6 +30,7 @@ __version__ = '1'
 class OutputFormat(Enum):
     raw_gpt = 1
     raw_btrfs = 2
+    raw_squashfs = 6
```

Comment:
I didn't want to change all the lines in the enum... But I figure it would be fine...